index.html

```diff
@@ -584,13 +592,18 @@ <h2>Terminology</h2>
 this document to other specifications. This specification decouples the
 [=identity provider=] concept into two distinct concepts: the [=issuer=]
 and the [=holder=].
+        </dd>
+        <dt><dfn class="export" data-lt="issuee|issuee's">issuee</dfn></dt>
+        <dd>
+A role an [=entity=] performs when it receives a [=verifiable credential=] from
+the [=issuer=].
         </dd>
```

Comment:
Does this overlap/repeat lines 376-382?
Does the `data-lt` attribute value (now snigular, plural, and possessive) need to include the (singular) string within the `<dfn>`?
```suggestion
        <dt><dfn class="export" data-lt="issue|issuees|issuee's">issuee</dfn></dt>
        <dd>
A role an [=entity=] performs when it receives a [=verifiable credential=] directly
from an [=issuer=].
        </dd>
```

Reply:
I am not sure the proposed change to the data-lt is correct. Lets discuss at a WG meeting.
Again, "directly" is not in the current text, so I prefer not to make this change.

Reply:
> I am not sure the proposed change to the data-lt is correct. Lets discuss at a WG meeting.

Fine.

> Again, "directly" is not in the current text, so I prefer not to make this change.

This (something which the WG needs to decide) seems more important to discuss in a WG call than the `data-lt` change (which is strictly a question of the technicalities of that attribute, and can be answered by anyone more versed in the finer points of ReSpec magic.)

Reply:
-1 to "directly" -- the meaning is vague. 

Is the use of a proxy to receive the credential "directly"? Is the use of a browser to receive it into a digital wallet "directly"? Is the use of a hold-and-pickup-later service "directly"? ... and if we explain what "directly" means, does it add value to the understand-ability of the text?

Reply:
OK. So, `issuee` is a role *assigned to* an entity (not *performed by* that entity, as no action on that entity's part is required), which assignment, made by the `issuer` of a VC, is achieved by including a `claim` that sets (a? the?) value of the `issuee` property to an identifier of that entity.

By that definition, transmission of the VC to that entity isn't required, which means that the `issuee` is *not* necessarily a `holder`.

(And there's enough hashing out happening through suggestions on this PR that the discussion should probably be moved to an issue or a non-suggestion thread, because the suggestions will vanish if merged, and become hard to see if resolved.)

Reply:
@TallTed -  Good suggestion. The issue is #1467 .  I have moved the proposed definition to this issue so that we can continue the discussion there and hence record all the arguments for posterity.